spec/adapters/postmark_adapter/outbound_spec.rb

```diff
@@ -288,5 +288,62 @@
         end
       end
     end
+
+    describe '::contributor_marked_as_inactive!' do
+      subject { described_class.contributor_marked_as_inactive!(admin, contributor) }
+
+      context 'no admin' do
+        let(:admin) { nil }
+        let(:contributor) { create(:contributor) }
+
+        it 'does not enqueue a Mailer' do
+          expect { subject }.not_to have_enqueued_job
+        end
+      end
+
+      context 'no contributor' do
+        let(:admin) { build(:user, admin: true) }
+        let(:contributor) { nil }
+
+        it 'does not enqueue a Mailer' do
+          expect { subject }.not_to have_enqueued_job
+        end
+      end
+
+      context 'user, not admin' do
+        let(:admin) { build(:user) }
+        let(:contributor) { create(:contributor) }
+
+        it 'does not enqueue a Mailer' do
+          expect { subject }.not_to have_enqueued_job
+        end
+      end
+
+      context 'admin email equals contributo marked inactive email' do
+        let(:admin) { build(:user, id: 231_123, admin: true, email: 'my-email@example.org') }
+        let(:contributor) { build(:contributor, id: 231_123, email: 'my-email@example.org') }
```

Comment:
```suggestion
        let(:admin) { build(:user, admin: true, email: 'my-email@example.org') }
        let(:contributor) { build(:contributor, email: 'my-email@example.org') }
```

Why the hard-coded id? I would expect it's not relevant? Users and contributors are two different models, or did I miss something?

Reply:
It was just to avoid creating the records. If we just build them, no ids are issued and that results in a serialization error.

Reply:
Ok, what about `create` instead of `build`?

I was just confused, I thought it has something to do with the feature itself.